### PR TITLE
Upgrading to Jython 2.7.2

### LINKED ISSUE #1743: Update Jython to 2.7.2
Ghidra uses Jython 2.7.1 which was released in 2017. 
Meanwhile Jython 2.7.2 is now available, released in March 2020.

https://www.jython.org/news.html

### LINKED ISSUE #1777: java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter
**Describe the bug**

I want to use a Ghidra script using [miasm](https://github.com/cea-sec/miasm) and I get the following error : 

```
script.py> Running...
Traceback (most recent call last):
  File "/home/user/ghidra_scripts/script.py", line 64, in <module>
    machine = Machine(cont.arch)
  File "/usr/local/lib/python2.7/dist-packages/miasm/analysis/machine.py", line 123, in __init__
    from miasm.arch.x86.disasm import dis_x86_64 as dis_engine
  File "/usr/local/lib/python2.7/dist-packages/miasm/arch/x86/disasm.py", line 2, in <module>
    from miasm.arch.x86.arch import mn_x86
java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter
	at org.python.compiler.Module.serializePyBytecode(Module.java:867)
	at org.python.compiler.Module.compile(Module.java:989)
	at org.python.core.imp.compileSource(imp.java:370)
	at org.python.core.imp.createFromSource(imp.java:388)
	at org.python.core.imp.loadFromSource(imp.java:667)
	at org.python.core.imp.find_module(imp.java:551)
	at org.python.core.PyModule.impAttr(PyModule.java:111)
	at org.python.core.imp.import_next(imp.java:840)
	at org.python.core.imp.import_logic(imp.java:905)
	at org.python.core.imp.import_module_level(imp.java:970)
	at org.python.core.imp.importName(imp.java:1057)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:450)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1149)
	at org.python.core.imp.importFrom(imp.java:1124)
	at miasm.arch.x86.disasm$py.f$0(/usr/local/lib/python2.7/dist-packages/miasm/arch/x86/disasm.py:29)
	at miasm.arch.x86.disasm$py.call_function(/usr/local/lib/python2.7/dist-packages/miasm/arch/x86/disasm.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:171)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromSource(imp.java:396)
	at org.python.core.imp.loadFromSource(imp.java:667)
	at org.python.core.imp.find_module(imp.java:551)
	at org.python.core.PyModule.impAttr(PyModule.java:111)
	at org.python.core.imp.import_next(imp.java:840)
	at org.python.core.imp.import_logic(imp.java:905)
	at org.python.core.imp.import_module_level(imp.java:970)
	at org.python.core.imp.importName(imp.java:1057)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:450)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1149)
	at org.python.core.imp.importFrom(imp.java:1124)
	at miasm.analysis.machine$py.__init__$2(/usr/local/lib/python2.7/dist-packages/miasm/analysis/machine.py:220)
	at miasm.analysis.machine$py.call_function(/usr/local/lib/python2.7/dist-packages/miasm/analysis/machine.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:171)
	at org.python.core.PyBaseCode.call(PyBaseCode.java:308)
	at org.python.core.PyBaseCode.call(PyBaseCode.java:199)
	at org.python.core.PyFunction.__call__(PyFunction.java:482)
	at org.python.core.PyMethod.instancemethod___call__(PyMethod.java:237)
	at org.python.core.PyMethod.__call__(PyMethod.java:228)
	at org.python.core.PyMethod.__call__(PyMethod.java:223)
	at org.python.core.Deriveds.dispatch__init__(Deriveds.java:19)
	at org.python.core.PyObjectDerived.dispatch__init__(PyObjectDerived.java:1112)
	at org.python.core.PyType.type___call__(PyType.java:1822)
	at org.python.core.PyType.__call__(PyType.java:1805)
	at org.python.core.PyObject.__call__(PyObject.java:480)
	at org.python.core.PyObject.__call__(PyObject.java:484)
	at org.python.pycode._pyx6.f$0(/home/user/ghidra_scripts/script.py:144)
	at org.python.pycode._pyx6.call_function(/home/user/ghidra_scripts/script.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:171)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.Py.runCode(Py.jva:1614)
	at org.python.core.__builtin__.execfile_flags(__builtin__.java:535)
	at org.python.util.PythonInterpreter.execfile(PythonInterpreter.java:286)
	at ghidra.python.GhidraPythonInterpreter.execFile(GhidraPythonInterpreter.java:232)
	at ghidra.python.PythonScriptExecutionThread.run(PythonScriptExecutionThread.java:51)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 58 more
java.lang.NoClassDefFoundError: java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter
````

I think I am bringing back #339 : `DatatypeConverter` is obsolete and removed in Java11 while it is still used by Jython 2.7.1. I have tried to use Java10 but Ghidra is just ignoring it and moving back to Java11. So I think the way to solve that on Ghidra side is to move to 2.7.2 ( #1743 ) as 2.7.2b1 seems to remove dependency on javax.xml.bind.DatatypeConverter. 

Do you see any way to have this running locally ? Would it be possible to run Ghidra temporarily with Java10 maybe ?

**To Reproduce**
1. Create a Ghidra script with miasm with the import `from miasm.arch.x86.arch import mn_x86`
2. Launch it in Ghidra
3. Crash

**Expected behavior**

The script should run.

**Environment (please complete the following information):**
 - OS: Debian sid
 - Java Version: openjdk version "11.0.7-ea" 2020-04-14
 - Ghidra Version: 9.1.2

### PR DESCRIPTION
**Pull request by ryanmkurtz**
Fixes #1743 
Fixes #1777

### CODE REVIEW

File: Ghidra/Features/Python/Module.manifest [hunk 1]
```diff
@@ -1,4 +1,4 @@
-DATA SEARCH IGNORE DIR: jython-2.7.0
+DATA SEARCH IGNORE DIR: jython-2.7.2
```
Comment (mumbel) [inline Ghidra/Features/Python/Module.manifest:1]: noticed this is jython-2.7.0 and not jython-2.7.1   was that supposed to be jython-2.7.1 this whole time

Comment (ryanmkurtz) [reply Ghidra/Features/Python/Module.manifest:1]: Yes, but it only affects efficiency.

**Comment by astrelsky** [comment]
I'm not sure if I should report this separately or not but I noticed something after integrating the changes locally but it appears that ghidra just takes the first version of jython it finds.




Shouldn't something like this be checked before it's loaded? I'm too lazy to attempt to hijack it at the moment but I think someone should check.

**Comment by ryanmkurtz** [comment]
This is a general jar + build issue so I think a new ticket would be appropriate.  For a variety of reasons, this is only an issue on custom user builds.

**Comment by ryanmkurtz** [comment]
@adamopolous, what do you think of this guy?

**Comment by ryanmkurtz** [comment]
It's kind of old, I can rebase it if you want

**Comment by adamopolous** [comment]
This one is pretty simple - I can put it through today

**Comment by ryanmkurtz** [comment]
Fixed in 2ef6a778b4d477657bc71b2ba44344df8d3cb2a1

**Pull request closed without merging**